sql/moz-fx-data-shared-prod/android_onboarding/android_funnel_draft_v1/query.sql

```diff
@@ -0,0 +1,391 @@
+-- extract the relevant fields for each funnel step and segment if necessary
+WITH onboarding_funnel_first_card AS (
+  SELECT
+    client_info.client_id AS join_key,
+    mozfun.map.get_key(extra, 'sequence_id') AS funnel_id,
+    onboarding_funnel_funnel_id.action AS action,
```

Comment:
not sure why these dimensions aren't being rendered according to their `select_expression` from the toml file